django/contrib/admindocs/templates/admin_doc/template_detail.html

```diff
@@ -5,8 +5,8 @@
 <div class="breadcrumbs">
 <a href="{% url 'admin:index' %}">{% translate 'Home' %}</a>
 &rsaquo; <a href="{% url 'django-admindocs-docroot' %}">{% translate 'Documentation' %}</a>
-&rsaquo; {% translate 'Templates' %}
-&rsaquo; {{ name }}
+&rsaquo; <a href="#">{% translate 'Templates' %}</a>
```

Comment:
It doesn't really make sense for this to be a link back to the current page, because it isn't...
```suggestion
&rsaquo; {% translate 'Templates' %}
```